Makefile

```diff
@@ -43,6 +43,10 @@ ifdef INSTALLATION_TIMEOUT
         INSTALLATION_TIMEOUT_FLAG = --installation-timeout $(INSTALLATION_TIMEOUT)
 endif
 
+ifdef ADMIN_USERS
```

Comment:
I don't think we need this here
For cloud deployments we'll get these values from the saas.yml
for non-cloud deployments we are using no-auth and then all users ar admin

@filanov , @YuviGold  what do you think?

Reply:
Agree